spec.md

```diff
@@ -1586,12 +1760,192 @@ en <= Z_valid
 cover(clk, pred, en, "X equals Y when Z is valid") : optional_name
 ```
 
+## Probes
+
+Probe references are created with `probe`{.firrtl} expressions, exported with
+the `export`{.firrtl} statement, forwarded between instances using the
+`forward`{.firrtl} statement, read using the `read`{.firrtl} expression (see
+[@sec:reading-probe-references]), and used with `force`{.firrtl} and
```

Comment:
```suggestion
[@sec:reading-probe-references]), and driven with `force`{.firrtl} and
```

not sure what you mean by "used"

Reply:
"Driven" is nice, thanks!  "Used" just meant "is valid as the reference operand of a force statement" and was indeed too vague on the result/mechanism of that use, thanks!